### PR TITLE
AG8-134-CY API -subscription

### PR DESCRIPTION
**Pull request by Cholpon3**

### CODE REVIEW

File: cypress/integration/step_definitions/backend/subscription.cy.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  expect(response.status).to.eq(200)
+
+})
+
+And('response should contain following data:', (dataTable) => {
+
+
+  const jsonData = cy.convertDatatableToJson(dataTable).then((jsonData) => {
+
+    jsonData.forEach((jsonItem, index) => {
```
Comment (KubraSA) [inline cypress/integration/step_definitions/backend/subscription.cy.js:30]: I think you don't need `index` parameter here.

File: cypress/support/commands.js [hunk 2]
```diff
@@ -22,4 +22,20 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('convertDatatableToJson', (dataTable) => {
```
Comment (KubraSA) [inline cypress/support/commands.js:27]: Very good custom command to have!

File: cypress/integration/step_definitions/backend/subscription.cy.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { And, Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+
+let response = null
```
Comment (KubraSA) [inline cypress/integration/step_definitions/backend/subscription.cy.js:3]: In js, you don't need to initialize value to as `null ` you can leave it as `let response`

Comment (Cholpon3) [reply cypress/integration/step_definitions/backend/subscription.cy.js:3]: Thank you for the review

**Review by KubraSA — APPROVED** [review]

**Pull request merged**